htdocs/includes/tcpdi/tcpdi_parser.php

```diff
@@ -484,7 +484,7 @@ protected function decodeXrefStream($startxref, $xref=array()) {
 			$v = $sarr[$key];
 			if (($key == '/Type') AND ($v[0] == PDF_TYPE_TOKEN AND ($v[1] == 'XRef'))) {
 				$valid_crs = true;
-			} elseif (($key == '/Index') AND ($v[0] == PDF_TYPE_ARRAY AND count($v[1]) >= 2)) {
+            } elseif (($key == '/Index') AND ($v[0] == PDF_TYPE_ARRAY AND count($v[1] >= 2))) {
```

Comment:
This looks strange
AND count($v[1] >= 2))) {
Old code
AND count($v[1]) >= 2)) {

Is it a fix on your own or official project patch ?

Reply:
it's come from here https://github.com/pauln/tcpdi_parser

Reply:
The old code was a fix when making lib compatible with php8. So i just think this fix was not included into the version at pauln. But i think we must keep it.
Can you try to also add this fix 
AND count($v[1]) >= 2)) {
And tell if your use of tcpdfi still works for the case you tested ?